src/account/kernel/constants/abis.ts

```diff
@@ -0,0 +1,5 @@
+export const accountAbi = [
+  'function installModule(uint256 moduleTypeId,address module,bytes calldata initData)',
+  'function uninstallModule(uint256 moduleTypeId,address module,bytes calldata deInitData)',
+  'function isModuleInstalled(uint256 moduleTypeId,address module,bytes calldata additionalContext) returns (bool)',
+]
```

Comment:
nit: we could just have a 7579 abi that is shared across the accounts and individual accounts could extend it with custom functions if needed